linen_examples/ppo/seed_rl_atari_preprocessing.py

```diff
@@ -32,8 +32,7 @@
 
 from gym.spaces.box import Box
 import numpy as np
-
-import cv2
+from cvx2 import latest as cv2
```

Comment:
can you confirm this is part of https://pypi.org/project/opencv-python/ ?
(from `requirements.txt`)

Reply:
Should have been reverted. Fixed.